lagunita/lms/static/sass/partials/base/custom_overrides/_course_styling.scss

```diff
@@ -154,3 +154,26 @@
         }
     }
 }
+
+#content {
+    .dashboard-banner {
+        .wrapper-msg.urgency-high {
+            background: #FFBF00;
+            position: relative;
+            z-index: 9;
```

Comment:
Is this necessary? I would think that if the HTML is placed in the right spot, it shouldn't need a z-index change?